src/index.html

```diff
@@ -9,6 +9,26 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Enclosures</h1>
+  <div class = "blue" id="qa-block" >
```

Comment:
```suggestion
  <div class = "blue-block" id="qa-block" >
```
you need to specify what is blue